src/util/getThoughtCommands.ts

```diff
@@ -0,0 +1,85 @@
+import CommandState from '../@types/CommandState'
+import FormattingCommand from '../@types/FormattingCommand'
+
+const commands = Object.values(FormattingCommand)
+
+/**
+ * This is a utility for creating opening and closing markup tag.
+ */
+const createTag = (tags: string) => ({
+  open: `<${tags}>`,
+  close: `</${tags}>`,
+})
+
+const tags = {
+  bold: createTag('b'),
+  italic: createTag('i'),
+  underline: createTag('u'),
+  strikethrough: createTag('strike'),
+}
+
+/**
+ * This determines which commands (bold, italic, underline, strikethrough)
+ * apply to an entire thought string.
+ */
+const getThoughtCommands = (thought: string): CommandState => {
```

Comment:
Let's rename the function (and file) `getThoughtCommands` → `getThoughtCommandState` or `getCommandState`, and rename the parameter `thought` → `value`.

`thought` is generally used throughout the codebase to refer to a variable of type [Thought](https://github.com/cybersemics/em/blob/b1192f02359bd524a9536000daecd227aecfc361/src/%40types/Thought.ts).

Reply:
I like `getCommandState`. Changed it here: df3a5a1be634ee8b51e073b89dd15e7bca17a8c1